lib/tasks/protractor.rake

```diff
@@ -67,6 +67,28 @@ namespace :protractor do |args|
     end
   end
 
+  desc "Run specs from config file"
+  task :server do
+    begin
+
+      webdriver_pid = fork do
```

Comment:
This is copy and pasted code. Could we not write it so that this code is reused in both use cases?